src/Extra/Pixlr.php

```diff
@@ -77,15 +83,24 @@ final class Pixlr
      * @param string $referrer
```

Comment:
Could not we declare these types and remove the docblocks?

Reply:
Yes, I plan to do that on another Pr , when phpstan level is strong enough to first tell if the phpdoc is correct.